### PR TITLE
Add env variable to control rate limiting

### PR DESCRIPTION
**Pull request by Ivo-Yankov**
**Description**:

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -15,4 +15,5 @@ TIER_3_RATE_LIMIT =
 LIMIT_DURATION =
 HBAR_RATE_LIMIT_TINYBAR =
 HBAR_RATE_LIMIT_DURATION =
+RATE_LIMIT_DISABLED =
```
Comment (Nana-EC) [inline .env.example:18]: Update the Helm chart also
See https://github.com/hashgraph/hedera-json-rpc-relay/pull/578/files as a reference

**Review by Nana-EC — CHANGES_REQUESTED** [review]
LG, needs chart updating also

**Review by Nana-EC — APPROVED** [review]
LG

**Review by georgi-l95 — APPROVED** [review]

**Review by georgi-l95 — CHANGES_REQUESTED** [review]
Just need to add this new variable to the [README.md](https://github.com/hashgraph/hedera-json-rpc-relay#testing)

**Review by georgi-l95 — APPROVED** [review]
LG

**Pull request merged**